src/test/java/org/springframework/data/couchbase/repository/support/SimpleReactiveCouchbaseRepositoryTest.java

```diff
@@ -0,0 +1,85 @@
+package org.springframework.data.couchbase.repository.support;
+
+import static org.mockito.ArgumentMatchers.*;
+import static org.mockito.Mockito.*;
+
```

Comment:
Please format your code as described in our [contribution guidelines](https://github.com/spring-projects/spring-data-build/blob/master/CONTRIBUTING.adoc), see [here for a formatter](https://github.com/spring-projects/spring-data-build/tree/master/etc/ide).